src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -76,6 +76,12 @@ const LoginForm: React.FunctionComponent<LoginFormProps> = ({
           "validation",
           `Error checking ticket: ${error.message}`
         );
+      } else if (error.code === "auth/user-not-found") {
+        setError(
+          "backend",
+          "firebase",
+          "Account not found. Please contact the event organizer."
+        );
```

Comment:
Same consideration as https://github.com/sparkletown/sparkle/pull/2283#commitcomment-56359981 and https://github.com/sparkletown/sparkle/pull/2283#commitcomment-56360170

Reply:
fixed in 6ee6ec5051d38171e2377f51df1516eba2969551